module/VuFindConsole/config/module.config.php

```diff
@@ -42,6 +42,7 @@
     ],
     'service_manager' => [
         'factories' => [
+            'VuFind\Sitemap\Generator' => 'VuFind\Sitemap\GeneratorFactory',
```

Comment:
@EreMaijala, this branch is not working for me, because this GeneratorFactory is missing. Did you neglect to commit the file to this branch? Thanks!